src/solvers/dg/containers_1d.jl

```diff
@@ -1,24 +1,27 @@
 
 # Container data structure (structure-of-arrays style) for DG elements
 # TODO: Taal refactor, remove NVARS, POLYDEG?
-mutable struct ElementContainer1D{RealT<:Real, NVARS, POLYDEG} <: AbstractContainer
-  inverse_jacobian::Vector{RealT}      # [elements]
-  node_coordinates::Array{RealT, 3}    # [orientation, i, elements]
-  surface_flux_values::Array{RealT, 3} # [variables, direction, elements]
-  cell_ids::Vector{Int}                # [elements]
+mutable struct ElementContainer1D{RealT<:Real, uEltype<:Real, NVARS, POLYDEG} <: AbstractContainer
```

Comment:
As a general question: Does it make sense to make our numerical types subtypes of `Number`  instead of `Real`? I was just wondering since we talked about allowing complex arguments for this Jacobian calculation trick mentioned in the Julia for HPC slack?

Reply:
OK, convinced. Also, we don't need it right now.